src/parser/FStar.Parser.ToDocument.fs

```diff
@@ -973,7 +973,7 @@ and p_aqual = function
     in
     str "#[" ^^ p_term false false t ^^ str "]" ^^ break1
   | Meta (Arg_qualifier_meta_attr t) ->
-    str "#[@" ^^ p_term false false t ^^ str "]" ^^ break1
+    str "#[@@" ^^ p_term false false t ^^ str "]" ^^ break1
```

Comment:
I think the "#[@" syntax is deprecated? Is that correct?

Reply:
Agreed, good to fix this. thanks.